### PR TITLE
Multis in a LevelGroup now show A,B,C,D

### PR DESCRIPTION
**Pull request by breville**
This is to correlate with the answer key used in teacher dashboard.  At some point it'd be nice to unify the radio buttons/checkboxes with these letters by using some new styling.

screenshot 2016-04-27 14 40 11
screenshot 2016-04-27 14 40 16
screenshot 2016-04-27 14 40 22

### CODE REVIEW

File: dashboard/app/views/levels/_single_multi.html.haml [hunk 1]
```diff
@@ -3,6 +3,8 @@
 - use_tight_layout = data['layout'] == "tight" || tight_layout
 - force_wrap_layout = data['layout'] == "wrap"
 
+- multi_answer_characters = ("A".."Z").to_a
```
Comment (tanyaparker) [inline dashboard/app/views/levels/_single_multi.html.haml:6]: In the basically not gonna happen chance that there are more than 26 options, what happens? Starts from the beginning?

**Comment by tanyaparker** [comment]
For the image multiple choice, is it possible to get the letter right next to the radio button too?

File: dashboard/app/views/levels/_single_multi.html.haml (hunk 1 above)
Comment (breville) [inline dashboard/app/views/levels/_single_multi.html.haml:6]: we'll just give an exception.  i think that's reasonable for such an edge case.

**Comment by breville** [comment]
Okay, updated to have the letter be separate to the answer item.

**Comment by tanyaparker** [comment]
woo! LGTM

**Pull request merged**